### ISSUE TITLE
fix: docker compose file is broken

### ISSUE THREAD

**Issue by Ushie** [revanced/revanced-releases-api#48]
**Describe the bug**
The DockerCompose file is incompatible with the rest of the project, the project expects an ENV value for UVICORN but receives ENVs for HYPERCORN, resulting with this terminal spam:

```
revanced-releases-api    | Usage: uvicorn [OPTIONS] APP
revanced-releases-api    | Try 'uvicorn --help' for help.
revanced-releases-api    | 
revanced-releases-api    | Error: Invalid value for '--port': '' is not a valid integer.
revanced-releases-api exited with code 2
```

**To Reproduce**
Build with the DockerCompose file

**Expected behavior**
Build goes smooth with no errors

**Comment by alexandreteles** [comment]
It is fixed by @Ushie on #51. I'm waiting for CI to finish it so we can merge everything into `dev`.

**Comment by alexandreteles** [comment]
Will be released with the merge of #53
